src/components/Listing/OffChain/OffChainPostsListing.tsx

```diff
@@ -25,7 +25,8 @@ const OffChainPostsListing: FC<IOffChainPostsListingProps> = ({ className, posts
 			</div>
 		);
 	}
-
+	//filter the posts that are deleted
+	posts = posts.filter((post) => post.isDelete !== true || post.isDelete === undefined);
```

Comment:
please double check this might cause pagination to break, if we use `startAt` or `startAfter`